src/com/nutrons/FRamework/StreamManager.java

```diff
@@ -0,0 +1,58 @@
+package com.nutrons.FRamework;
+
+import io.reactivex.Flowable;
+import io.reactivex.Observable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class sets up the I/O factories and initializes subsystems.
+ */
+public class StreamManager {
+  private List<Subsystem> subsystems;
+  private Flowable<Boolean> enabled;
+  private Flowable<CompMode> mode;
+
+  /**
+   * Manages subsystems and the competition loop.
+   * @param enabled a Flowable of booleans, representing changes in the enabled state of the robot
+   * @param mode a Flowable of CompModes, representing changes in the competition game mode
+   */
+  public StreamManager(Flowable<Boolean> enabled, Flowable<CompMode> mode) {
```

Comment:
Are these necessary as arguments, or could they be accessed statically or initialized internally?